compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -2710,7 +2708,8 @@ object Parsers {
         if (in.token == PACKAGE) {
           in.nextToken()
           if (in.token == OBJECT) {
-            ts += objectDef(start, atPos(start, in.skipToken()) { Modifiers(Package) })
+            in.nextToken()
```

Comment:
Why not `skipToken`?

Reply:
We don't need the old position returned by skipToken()